expression/builtin_encryption.go

```diff
@@ -25,10 +25,13 @@ import (
 	"fmt"
 	"hash"
 	"io"
+	"strings"
 
+	"crypto/aes"
```

Comment:
please put the standard packages together and add an empty line between the third-part packages.

Reply:
done.